src/Schema/Builder.php

```diff
@@ -137,9 +142,10 @@ public function dropAllTables()
         }
     }
 
-    public function getTables()
+    /** @param string|null $schema Database name */
+    public function getTables($schema = null)
```

Comment:
Are the signature changes here also a potential BC break for Laravel 11 users, or would you not expect anyone to extend this class?

Reply:
Yes, this is a breaking change if someone extends our class... I hope nobody does. We should make more classes final.